locust/runners.py

```diff
@@ -279,6 +279,15 @@ def on_locust_switch_file(locust_classes_key):
                 self.server.send(Message("switch", locust_classes_key, None))
         events.locust_switch_file += on_locust_switch_file
 
+        def on_master_new_configuration(new_config):
+            logger.info("report slaves to update their config")
+            data =  {
```

Comment:
any consideration to passed it on dictionary? you can pass the new_config directly

Reply:
yes, you can pass it directly, but I love to pass it as a dictionary instead. it's more readable in every function who use the data from this message

Reply:
yea thats nice, just make sure my point is about memory usage since we send it by network from master to slave via socketrpc, its adding 280 byte if used dictionary, wdyt?

Reply:
it doesn't affect anything to the load test, it doesn't called when we start the load test, and I can say it takes 0% percent CPU usage since it's not a complex computation like regex things

Reply:
also the memory taken for that variable cleared on function exit